client/web/ui_components/app-root.js

```diff
@@ -608,6 +608,30 @@ export class AppRoot extends mixinBehaviors(
   ready() {
     super.ready();
     this.setLanguage(this.language);
+    this.addEventListener('ShowNavigation', () => {
+      this.openDrawer();
+    });
+    this.addEventListener('HideNavigation', () => {
+      this.closeDrawer();
+    });
+    this.addEventListener('ShowAddServerDialog', () => {
+      if (this.$.addServerView) {
+        this.$.addServerView.open = true;
+      }
+    });
+    this.addEventListener('AddServerRequested', () => {
+      if (this.$.addServerView) this.$.addServerView.open = false;
+    });
+    this.addEventListener('IgnoreServerRequested', () => {
+      if (this.$.addServerView) this.$.addServerView.open = false;
+    });
+    this.addEventListener('ChangePage', event => {
+      const detail = /** @type {CustomEvent} */ (event).detail || {};
+      if (detail.page) {
+        this.changePage(detail.page);
+      }
+      this.closeDrawer();
+    });
```

Comment:
We have already handled these events: https://github.com/nthlink/outline-apps/blob/311139e2791c945e53bd42fe2a4e65b951d88719/client/web/app/app.ts#L138-L141

Please investigate why they are not fired properly from here: https://github.com/nthlink/outline-apps/blob/311139e2791c945e53bd42fe2a4e65b951d88719/client/web/views/root_view/root_header/index.ts#L72-L77

Please revert.

Reply:
Verified Ubuntu build.